test/navigation/nonsipp/EmployerContributionsNavigatorSpec.scala

```diff
@@ -231,4 +233,16 @@ class EmployerContributionsNavigatorSpec extends BaseSpec with NavigatorBehaviou
     )
   }
 
+  "EmployerContributionsCYAPage" - {
+    act.like(
+      normalmode
+        .navigateTo(
+          EmployerContributionsCYAPage,
+          (srn, _) =>
+            controllers.nonsipp.employercontributions.routes.EmployerContributionsMemberListController
+              .onPageLoad(srn, page = 1, NormalMode)
+        )
+        .withName("go from EmployerContributionsCYAPage to ??? page")
```

Comment:
EmployerContributionsMemberList instead of ???

Reply:
good shout, will add it onto my next ticket